src/ui/tiling-background.ts

```diff
@@ -0,0 +1,27 @@
+import { HEIGHT, WIDTH } from "../main";
```

Comment:
Maybe reuse this for the title screen

Reply:
wdym?
Title screen is a placeholder will be custom art by @Arekkus
HEIGHT and WIDTH get used everywhere instead of hard coded values?